.github/workflows/ci.yml

```diff
@@ -157,7 +157,7 @@ jobs:
     - name: make
       run: make -s -j4
     - name: make test
-      run: make TESTS=test_threads test HARNESS_JOBS=${HARNESS_JOBS:-4}
+      run: make V=1 TESTS="test_threads test_internal_provider test_provfetch test_provider test_pbe test_evp_kdf test_pkcs12 test_store test_evp" test HARNESS_JOBS=${HARNESS_JOBS:-4}
```

Comment:
The tests being run here were just the ones that produced sanitiser output.  We should run as many of the tests as is reasonable given time and memory constraints.  I.e. the list should be expanded.  This isn't something that needs to be done here.

I'm seeing test failures locally.  Looks like these ones which aren't failing in the CI loops.